### PR TITLE
feat: azure oauth2 login configuration

### PR DESCRIPTION
**Pull request by urfreespace**

### CODE REVIEW

File: charts/sn-console/values.yaml [hunk 1]
```diff
@@ -118,6 +125,13 @@ login:
       config:
         GOOGLE_RESTRICT_DOMAIN_SUFFIXES_NAME: ""
         GOOGLE_REDIRECT_URI: ""
+    azure:
+      enabled: false
+      # The secret should contain both `GOOGLE_CLIENT_ID` and `GOOGLE_CLIENT_SECRET`
+      clientSecretRef: '[kubernetes secret that stores client secret for Google SSO]'
```
Comment (tuteng) [inline charts/sn-console/values.yaml:131]: Move Google SSO to `Azure AD SSO`

**Review by tuteng — COMMENTED** [review]
https://github.com/streamnative/charts/tree/master/charts/sn-platform/templates/streamnative-console Please update this module also.

File: charts/sn-console/values.yaml (hunk 1 above)
Comment (urfreespace) [reply charts/sn-console/values.yaml]: fixed

**Review by tuteng — APPROVED** [review]

**Comment by Huanli-Meng** [comment]
The doc is updated. Here is the [PR](https://github.com/streamnative/doc-platform/pull/381). Therefore, remove the doc-required label and add doc-added label.

**Pull request merged**